src/Framework/PaymentGateways/PaymentGateway.php

```diff
@@ -307,4 +310,59 @@ private function handleExceptionResponse($exception, $message)
         give_set_error('PaymentGatewayException', $message);
         give_send_back_to_checkout();
     }
+
+    /**
+     * @unreleased
+     *
+     * @param string $method
+     *
+     * @return bool
+     */
+    public function supportsMethodRoute($method)
```

Comment:
Looks like you're just not using the trait methods at all. You can rename the trait methods and then call them in here if the subscription module doesn't have the method.

Reply:
@JasonTheAdams Nope. We are using this function in `GatewayRoute::__invoke`

Reply:
Adjusted in 26d1b462f